docs/coding-guidelines/project-guidelines.md

```diff
@@ -16,7 +16,7 @@ Below is a list of all the various options we pivot the project builds on:
 
 - **Target Frameworks:** .NETFramework, .NETStandard, .NETCoreApp
 - **Platform Runtimes:** .NETFramework (aka CLR/Desktop), CoreCLR, Mono
-- **OS:** Windows_NT, Linux, OSX, FreeBSD, AnyOS
+- **OS:** windows, Linux, OSX, FreeBSD, AnyOS
```

Comment:
Why make Windows be lowercase but the rest capitalized? Is the plan to make the rest lowercase too?

Reply:
yes the plan is to make them similar to sdk targetPlatform strings